kornia/contrib/models/rt_detr/model.py

```diff
@@ -167,10 +233,10 @@ def forward(self, images: Tensor) -> tuple[Tensor, Tensor]:
               :math:`K` is the number of classes.
             - **boxes** - Tensor of shape :math:`(N, Q, 4)`, where :math:`Q` is the number of queries.
         """
-        if self.training:
-            raise RuntimeError("Only evaluation mode is supported. Please call model.eval().")
+        # if self.training:
```

Comment:
remove ? or put comment if we want to support that later

Reply:
IDK why we put it. It can be optimised if I write the boiler code by hand.

Reply:
Then remove it